### DISCUSSION TITLE
Fundamentos do SQL Server > Backup e Restore

### DISCUSSION THREAD

**Discussion by borgestech** [balta-io/forum#633] [Tira Dúvidas] [answered]
Ao importar o balta.bacpac estou tendo esse erro:
![Screenshot_1](https://github.com/user-attachments/assets/a2db8f7b-6b75-4599-af59-eed345e5f74f)

Obs.: Segui todos os passos da aula...

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @borgestech, como vai?

Provavelemente sua base está em uso! Tenta parar todos os serviços (Até reiniciar a máquina) e fazer só o import, sem nada rodando na base.
